resources/webviews/periscope/periscope.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AKS Periscope</title>
+    <!-- Link to the css file -->
+    <link rel="stylesheet" href="{{cssuri}}">
+    <script src="https://use.fontawesome.com/7dddd54b5c.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+
+    <script>
+        const vscode = acquireVsCodeApi();
+
+        function generateDownloadLink() {
+            // $("input").trigger("click");
+            vscode.postMessage({ command: "default" })
+        }
+        function generateSevenDaysDownloadLink() {
+            vscode.postMessage({ command: "7days" })
+        }
+    </script>
+</head>
+
+<body>
+
+    <div class="heading">
+        <h2 class="content-header">Periscope ran for cluster: {{name}}</h2>
+        <br\> Know more about periscope <a href="https://aka.ms/periscope">https://aka.ms/periscope</a>
+    </div>
+
+    <div class="link">
+    </div>
+    <hr />
+
+
+    <div class="insight-container">
+        <div class="panel panel-default data-container">
+            <div class="panel-heading data-container-heading">
+                <h5 class="panel-title">AKS Periscope upload logs information.</h5>
+            </div>
+        {{#unless outputCode}}
+            <div class="panel-body data-container-body">
+                Your logs are being uplaoded to the storage account: {{storageAccName}}.
+            </div>
+
+            <div class="panel-body data-container-body">
+                <h5 class="panel-title">AKS periscope the logs are uploaded successfully, <br />
+                    (Please note: Due to the nature how Periscope tool works it could be a possibility that log files of
+                    some nodes are still writing to the storage hence
+                    it is recommended to re-click generate button or wait for a minutebefore generating the downloable link.
```

Comment:
💡 Yeah, so essentially this is `by design` for AKS for now, but this will change: https://github.com/Azure/aks-periscope/issues/30  I have bunch of other enhancement mentioned against this repository and planning to latter possibly float a design doc for the `aks-periscope` what happens is that `aks-periscope` tool runs, and whole storage is still loading the logs, it goes greedy and reports, I am done, (essentially not actually done, runs the process at the background, i.e. no `go- channel` under the hood.